python/mxnet/contrib/quantization.py

```diff
@@ -978,7 +1004,7 @@ def __exit__(self, exc_type, exc_value, traceback):
         net.collect_params().reset_ctx(ctx)
     return net
 
-def quantize_net(network, quantized_dtype='auto', quantize_mode='full',
+def quantize_net(network, quantized_dtype='auto', quantize_mode='full', quantize_granularity='tensor-wise',
```

Comment:
it's better to keep.

Reply:
Looks like the only difference between `quantize_net_v2` and `quantize_net` is the new param `LayerOutputCollector` so far, `quantize_granularity` will be another new param introduced by this PR.
As there's not any other difference, how about jsut combine these two API into one?
Otherwise, there could be `v3`, `v4` version in the future if there's new param needed.

Reply:
I don't know if there is a better way to handle this kind of change. Since `quantize_net` is an user interface and has been released, adding new parameters to it might break user workloads. One convention is to add `v2` implementation, deprecate the original one, and replace it in the next major release. The `v2` implementation is not released yet, so I think it's safe to change it to add `quantize_granularity`. If it's released, yes, we might need `v3` implementation.

Reply:
That make sense, so let's keep the `quantize_net` api as is, only adding the new params in `v2` version.